components/content/src/utils.rs

```diff
@@ -8,6 +8,13 @@ use utils::fs::is_temp_file;
 use utils::table_of_contents::Heading;
 
 pub fn has_anchor(headings: &[Heading], anchor: &str) -> bool {
+    // HTML specs state a special anchor: "top" (case-insensitive)
+    // which is used to refer to the start of the page
+    // https://html.spec.whatwg.org/#select-the-indicated-part
+    if anchor.eq_ignore_ascii_case("top") {
```

Comment:
Is it the right place to do it? This checks for the anchors in the heading, it feels like this check should be before calling that function.

Reply:
updated!